client/src/index.css

```diff
@@ -1,13 +1,19 @@
+html {
+  height: 100%;
+}
 body {
+  height: 100%;
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
-
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+#root {
+  height: 100%;
+}
```

Comment:
What are you trying to do here with all these `height: 100%`? Does the `body` or `root` element need to take up the whole screen? If so, then you should put `min-height: 100vh` on one of them (whichever one makes the most sense)